keyboards/sofle/keymaps/nartudy/config.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#ifndef NO_DEBUG
+#define NO_DEBUG
+#endif // !NO_DEBUG
+#if !defined(NO_PRINT) && !defined(CONSOLE_ENABLE)
+#define NO_PRINT
+#endif // !NO_PRINT
```

Comment:
These aren't needed.  If the console isn't enabled, they're defined already.  If the console is enabled, then you probably don't want them defined. 
```suggestion
```